dcos_launch/config.py

```diff
@@ -162,17 +182,15 @@ def get_validated_config(user_config: dict, config_dir: str) -> dict:
 
 
 COMMON_SCHEMA = {
-    'deployment_name': {
-        'type': 'string',
-        'required': True},
     'provider': {
         'type': 'string',
         'required': True,
         'allowed': [
             'aws',
             'azure',
             'acs-engine',
-            'onprem']},
+            'onprem',
+            'terraform']},
```

Comment:
nitpick: add trailing comma and put ending brackets on new line.

```
  'terraform',
]}
```

Reply:
Done